how-to/deploy/orchestrated/tiup.md

```diff
@@ -1345,7 +1346,7 @@ ID              Role          Host      Ports                            Status
 
 #### 查看 TiDB Dashboard 检查 TiDB 集群状态
 
-- 通过 `{pd-leader-ip}:2379/dashboard` 登录 TiDB Dashboard，登录用户和口令为 TiDB 数据库 root 用户和口令，如果你修改过数据库的 root 密码，则以修改后的密码为准，默认密码为空。
+- 通过 `{pd-ip}:2379/dashboard` 登录 TiDB Dashboard，登录用户和口令为 TiDB 数据库 root 用户和口令，如果你修改过数据库的 root 密码，则以修改后的密码为准，默认密码为空。
```

Comment:
To make the commit from https://github.com/pingcap/docs-cn/pull/2919 in `master` branch